lib/screenplay_web/controllers/pa_messages_api_controller.ex

```diff
@@ -49,11 +52,28 @@ defmodule ScreenplayWeb.PaMessagesApiController do
 
   def show(conn, %{"id" => id}) do
     if pa_message = PaMessages.get_message(id) do
-      json(conn, pa_message)
+      {:ok, template} = StaticTemplates.get_template(pa_message.template_id)
+      alert = AlertsCache.alert(pa_message.alert_id)
+
+      json(conn, %{
+        pa_message: pa_message,
+        template: template,
+        alert: if(is_nil(alert), do: alert, else: Alert.to_full_map(alert))
+      })
     else
       conn
       |> put_status(404)
       |> json(%{error: "not_found"})
     end
   end
+
+  def static_templates(conn, _) do
```

Comment:
Apologies if this was already discussed, but do we need to serve these from the backend? I was imagining just compiling the template text into the frontend bundle and accessing it statically, since that's the only place we need it.

Reply:
I don't think server vs. client was ever discussed, but that's an interesting idea. It didn't click until now that the server has no reason to care about the contents of the file. That should simplify some things so I can give that a shot and see how we feel.